src/api/todos.ts

```diff
@@ -1,8 +1,22 @@
 import { Todo } from '../types/Todo';
 import { client } from '../utils/fetchClient';
 
-export const getTodos = (userId: number) => {
+export const getTodos = (userId: number | undefined) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
-// Add more methods here
+export const createTodo = (userId: number | undefined, query: string) => {
```

Comment:
There is probably no case to get `undefined` as `userId` and we'll never send a post request with `undefined`, but it's better to prevent it, even theoretically. You should check if userId exists before calling this function (if not - do nothing)